spec/models/sponsor_spec.rb

```diff
@@ -328,6 +328,24 @@
       expect(request_fulfilled_sponsor.eligible_for_sponsorship?).to eq false
     end
 
+    describe '#sponsorship_changed!', :mytest => true do
+
+      before(:each) do
+        allow(active_sponsor).to receive(:update_request_fulfilled!)
+        allow(active_sponsor).to receive(:update_active_sponsorship_count!)
+        active_sponsor.sponsorship_changed!
+      end
+
+      it 'should call #update_request_fulfilled!' do
+        expect(active_sponsor).to have_received(:update_request_fulfilled!)
+      end
+      
+      it 'should call #update_active_sponsorship_count!' do
+        expect(active_sponsor).to have_received(:update_active_sponsorship_count!)
+      end
```

Comment:
The two `allow` stubs should also then be moved into this single method (since there is no point in `before(:each)` anymore? 

    specify '#sponsorship_changed! should call update methods' do
        allow(active_sponsor).to receive(:update_request_fulfilled!)
        allow(active_sponsor).to receive(:update_active_sponsorship_count!)
        active_sponsor.sponsorship_changed!

      expect(active_sponsor).to have_received(:update_request_fulfilled!)
      expect(active_sponsor).to have_received(:update_active_sponsorship_count!)
end